### PR TITLE
Update contracts to v8

### PR DESCRIPTION
**Pull request by pinosu**

### CODE REVIEW

**Review by maurolacy — APPROVED** [review]
LGTM.

File: tests/testdata/mesh_vault.wasm.gz
Comment (maurolacy) [inline tests/testdata/mesh_vault.wasm.gz:1]: The vault contract changed instantiation message in the version.

Curious that this is not making some tests fail.

Comment (alpe) [reply tests/testdata/mesh_vault.wasm.gz:1]: tests were not executed as this PR is not targeting main, I guess

**Review by alpe — APPROVED** [review]
👍

**Pull request merged**